PageViewController.swift

```diff
@@ -0,0 +1,211 @@
+//
+//  PageViewController.swift
+//  berkeley-mobile
+//
+//  Created by Eashan Mathur on 10/17/20.
+//  Copyright © 2020 ASUC OCTO. All rights reserved.
+//
+
+import UIKit
+
+class PageViewController: UIPageViewController, UIPageViewControllerDataSource, UIPageViewControllerDelegate {
+    
+    
+    var pages = [UIViewController]()
+    private let pageControl : UIPageControl = {
+        let pageControl = UIPageControl()
+        return pageControl
+    }()
+    var currentIndex: Int {
+        guard let vc = viewControllers?.first else { return 0 }
+        return pages.firstIndex(of: vc) ?? 0
+    }
+    var getStarted = UIButton()
+    let skipButton = UIButton()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.view.backgroundColor = .white
+        
+        self.dataSource = self
+        self.delegate = self
+
+        configurePageControl()
+
+        configureButtons()
+    }
+    
+    override func viewDidLayoutSubviews() {
+        pageControl.subviews.forEach {
+            $0.transform = CGAffineTransform(scaleX: 1.5, y: 1.5)
+        }
+    }
+    
+    func configureButtons() {
+        print("Button should show")
+        let nextButton = UIButton()
+        nextButton.setTitle("Next", for: .normal)
+        nextButton.setTitleColor(.gray, for: .normal)
+        nextButton.frame = CGRect(x: 30, y: 100, width: 50, height: 30)
+        view.addSubview(nextButton)
+        nextButton.addTarget(self, action: #selector(nextClicked(_:)), for: .touchUpInside)
+        nextButton.translatesAutoresizingMaskIntoConstraints = false
+        nextButton.centerYAnchor.constraint(equalTo: pageControl.centerYAnchor).isActive = true
+        nextButton.rightAnchor.constraint(equalTo: view.safeAreaLayoutGuide.rightAnchor, constant: -24).isActive = true
+        nextButton.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor).isActive = true
+        
+        
+        skipButton.setTitle("Skip", for: .normal)
+        skipButton.setTitleColor(.gray, for: .normal)
+        skipButton.frame = CGRect(x: 30, y: 100, width: 50, height: 30)
```

Comment:
generally don't know if it's a good idea to mix setting the frame and setting constraints. it looks like the x/y for the button are overridden anyway so the frame is just used to set the width/height. in that case, just setting width/height using constraints should achieve the same effect.
same for the other buttons